src/pages/home/HomePage.tsx

```diff
@@ -63,6 +64,10 @@ export default class HomePage extends React.Component<IHomePageProps, IHomePageS
 
         exposeComponentRenderer('renderQuerySelector', ()=>{ return <QueryAndDownloadTabs store={this.store} />  });
 
+        exposeComponentRenderer('renderQuerySelectorWithBlurb', ()=> { 
```

Comment:
We shouldn't call render method this way.   We can just use the  reactRoot, which you've done.  It's possible we're hiding the react root in index.jsp, but we don't need to anymore.  I can explain.